ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/responsiveGrid.html

```diff
@@ -22,6 +22,8 @@
          data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"
          title="${panel.tooltipVisible ? '' : panel.tooltipText}"
          role="${container.roleAttribute}"
+         data-sly-test.placeholderText="${'Please drag {0} components here' @ i18n, format=[panel.label.value]}"
+         data-placeholder-text="${panel.label.value ? placeholderText : 'Drag Panel components here'}"
```

Comment:
How will localization work for dynamic labels ? We would have to document this.

Both the strings have to be `Granite.I18n.get`

Can you make use of

Reply:
label is generally according to the type of form field from an user persistence.
if the label of the panel is "Enter you address details", then would we say: 
"Please drag Enter you address details components here"
I don't think it would make sense.
I'd still prefer to use "'Drag Panel components here" irrespective of the label.

Reply:
@amanthakur03 , i think we need to discuss the case nitin is referring to before merging this

Reply:
Yes Deep, Posting on slack for everyone's suggestions.

Reply:
> I'd still prefer to use "'Drag Panel components here" irrespective of the label.

Should we rather use name here ? The problem with name is that, sometimes business author don't change the auto generated name, unless they use it in rules.

Reply:
> I'd still prefer to use "'Drag Panel components here" irrespective of the label.
That gets confusing when you have nested hierarchy. `Drag Panel components(Title) here` or on hover show the title.